skynet/modules/ttt/rag/vector_store.py

```diff
@@ -105,26 +117,39 @@ async def get_config(self, store_id: str) -> Optional[RagConfig]:
 
         return None
 
-    async def workflow(self, payload: RagPayload, store_id: str):
+    async def workflow(self, store_id: str, files: list[str], urls: list[str], max_depth: int):
         """
-        Crawl the given URL and create a vector store with the generated embeddings.
+        Extract text from the payload and create a vector store with the generated embeddings.
         """
 
+        documents = []
+        error = None
+
         try:
-            documents = await crawl(payload)
+            from skynet.modules.ttt.rag.text_extractor.main import extract
```

Comment:
Any reason to import these here rater than at the top?

Reply:
not anymore